windows/guest_customization/handle_guest_known_issues.yml

```diff
@@ -8,17 +8,23 @@
 # https://kb.vmware.com/s/article/87307
 - name: "Workaround for SYSPREP error caused by Appx package"
   block:
-    - name: "Remove BingSearch Appx package in Windows Client"
+    - name: "Remove Appx packages in Windows Client"
       include_tasks: ../utils/win_remove_appx_package.yml
       vars:
-        win_appx_package: "BingSearch"
+        win_remove_appx_ignore_errors: true
```

Comment:
If the packages are not removed, gosc will fail. Why ignore error here?

Reply:
When the issue happens this is a workaround, while the appx packages will not always cause GOSC failure, so this is a try best task.